package.json

```diff
@@ -6,7 +6,7 @@
   "typings": "./lib/index.d.ts",
   "scripts": {
     "lint": "eslint .",
-    "build": "tsc src/*.ts -d --strictNullChecks --sourceMap --outdir ./lib --lib ES6,ES5",
+    "build": "tsc src/*.ts src/**/*.ts -d --strictNullChecks --sourceMap --outdir ./lib --lib ES6,ES5",
```

Comment:
@sveisvei any fancy ant-path expressions we can use instead here, this looks a bit verbose...?